crossdock/server/tch/server.go

```diff
@@ -65,7 +65,9 @@ func Stop() {
 func register(ch *tchannel.Channel) {
 	ch.Register(raw.Wrap(echoRawHandler{}), "echo/raw")
 
-	json.Register(ch, json.Handlers{"echo": echoJSONHandler}, onError)
+	json.Register(ch, json.Handlers{
+		"echo": echoJSONHandler,
+	}, onError)
```

Comment:
this spacing is funny, multiple code reviews have caused us to flip it back and forth :)